src/openrct2/libopenrct2.vcxproj

```diff
@@ -1085,6 +1085,7 @@
     <ClCompile Include="Version.cpp" />
     <ClCompile Include="windows\Intent.cpp" />
     <ClCompile Include="world\tile_element\EntranceElement.cpp" />
+    <ClCompile Include="world/tile_element/TileElementBase.cpp" />
```

Comment:
CI didn't seem to care and built anyway, but most of the other places are using `\` over `/`